public/js/chat-cmds.js

```diff
@@ -213,3 +209,103 @@ var DashboardToggle = function() {
     return this;
 };
 
+var ChatCommands = {
+    _cmds: {},
+
+    _chatObject: null,
+
+    /**
+     * Initializes the chat commands object.
+     *
+     * @param {Object} chatObj - Chat object.
+     * @returns {undefined} undefined
+     * @public
+     */
+    initialize: function(chatObj) {
+        this._chatObject = chatObj;
+
+        // @todo Add new plugins here
+        this.registerCommand(new ListUser());
+        this.registerCommand(new CameraToggle());
+        this.registerCommand(new MicToggle());
+        this.registerCommand(new DashboardToggle());
```

Comment:
As mentioned above, moved these definitions to the top so they're available here. (They were being hoisted before)